articles/references/core_modules/toc.yml

```diff
@@ -0,0 +1,11 @@
+- name: Intent Common
+  href: intent_common/overview.md
+- name: Intent Common Types
+  href: intent_common_types/overview.md
+- name: Intent Nuget Installer
+  href: intent_nuget_installer/overview.md
+- name: Intent Visual Studio Projects
+  href: intent_visual_studio_projects/overview.md
+- name: Roslyn Weaver
+  href: roslyn_weaver/toc.yml
+  topicHref: roslyn_weaver/overview.md
```

Comment:
For maintainability purposes, is it possible for the `name`s to be blank and have DocFX automatically populate it with the article title from an `xref`? This may require that we change TOC files from `.yml` to `.md`.